examples/dom-bindings/src/App.js

```diff
@@ -23,32 +23,34 @@ class App extends Component {
           <h2>Welcome to React PDF</h2>
         </div>
         <p className="App-intro">
-          <Document height="100%" width="100%">
-            <Page size="A4">
-              <View style={styles.container}>
-                <Text>
-                  Text
-                </Text>
-                <View style={styles.block}>
+          <Pdf height="100%" width="100%">
+            <Document>
```

Comment:
Still choose to keep the Document component in react-pdf, because it is still usefull in every renderer. But the react-pdf-dom component renders now the Document component from the react-pdf package, and only omits the react-dom specific properties.